controllers/apply/building-connections/fields.js

```diff
@@ -2,7 +2,7 @@
 const { check } = require('express-validator/check');
 
 const LENGTH_HINTS = {
-    ONE_LINER: {
+    HEMINGWAY: {
```

Comment:
We never use this length but I couldn't resist.

Reply:
💯